docs/CHANGELOG.md

```diff
@@ -0,0 +1,90 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+### Added
+ - Add validator for `era5cli.txt` keys. This should provide better feedback to users and reduce user error.
+ - Added --splitmonths argument for `era5cli hourly`. This allows users to avoid a Request Too Large error.
+
+### Changed
+ - Change CDS keys from `.cdsapirc` file to `.config/eracli.txt` file. This will avoid conflict with e.g. ADS.
+ - If a user makes a request without `--splitmonths` they are warned that the behavior will change in the future, and that they have to choose between `--splitmonths False` and `--splitmonths True`.
+ - When a request would encounter a Request Too Large error in the CDS API, they are warned, and given a suggestion to use `--splitmonths`.
+ - `cli.py` has been refactored to make the structure more clear. Seperate argument builders are now in their own modules.
+
+
+## [1.3.2] - 2021-12-13
+### Changed
+ - Elaborate the range of years that can be queried [#123](https://github.com/eWaterCycle/era5cli/pull/123)
```

Comment:
I feel like the sub-menu on the right-hand side of the page has too many levels on the changelog page currently:

![image](https://user-images.githubusercontent.com/17080502/226607426-28cc6cbd-d718-44ac-8a9c-429ccffb3c17.png)